### DISCUSSION TITLE
What's the difference between `karpenter.sh/do-not-consolidate` and `spec.consolidation.enabled`?

### DISCUSSION THREAD

**Discussion by ashrafguitoni** [kubernetes-sigs/karpenter#1038] [Q&A]
The docs contain the following statement:
> Provisioner .spec.annotations allow you to set annotations that will be applied to all nodes launched by this provisioner. By setting the annotation karpenter.sh/do-not-consolidate: "true" on the provisioner, you will selectively prevent all nodes launched by this Provisioner from being considered in consolidation calculations.

Could be that I didn't understand due to english not being my native language, but I still feel like it's not so clear anyways. Could someone please provide a simpler comparison or explanation of how `karpenter.sh/do-not-consolidate` is different?

**Comment by jfoechsler** [comment]
Does it have to be two different things? Seems more like a case of two options for how to disable it. I guess its now via `spec.disruption.consolidateAfter: Never ` instead of the one you mentioned - Correction, the docs says consolidateAfter: Never only applies to empty node consolidation.
